src/main/java/com/BrassAmber/ba_bt/entity/block/MonolithEntity.java

```diff
@@ -67,8 +68,13 @@ public MonolithEntity(EntityType<? extends MonolithEntity> type, World world) {
 		this.correctGuardianEye = GolemType.getEyeFor(GolemType.getPreviousGolemType(this.golemType));
 	}
 
-	public MonolithEntity(EntityType<MonolithEntity> monolithEntityType, World worldIn, double x, double y, double z) {
+	public MonolithEntity(EntityType<MonolithEntity> monolithEntityType, World worldIn, double x, double y, double z, BlockState placedOnState) {
 		this(monolithEntityType, worldIn);
+		if (placedOnState == Blocks.CLAY.defaultBlockState()) {
```

Comment:
Is this testing purposes?

Reply:
Yes, I wanted to be able to still place the monolith as if it was generated by the structure, but make it so that regular modapack:map makers could still place it wherever without triggering the Destroy tower Entity. 
This is also for later down the line when I need to add the monoliths to say the ocean or nether tower.

Reply:
I also removed being able to do /summon on Monoliths so that players don't accidentally create giant pits when they are just testing fighting the golem.